00-HTML-CSS-basics/CSS/Basics/css/style.css

```diff
@@ -0,0 +1,52 @@
+
+/*font styles*/
+/*----------*/
+
+@font-face{
+  font-family: 'MyFont';
+  src:url('../fonts/amatic/Amatic-Bold.ttf');
+  font-weight: bold;
+  font-style: normal;
+  font-size: 14px;
+ 
+}
+
+/*element selectors*/
+/*----------------*/
+html{
+  font-family: MyFont, Ubuntu, Arial, TimesNewRoman, sans-serif;
+}
+
+header,footer{
```

Comment:
Also why you use html tags as selectors intead of classes? As a good practice we should use in most cases just classes.